clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -1760,24 +1767,37 @@ def validate_args(argv, command, expected_nr):
         raise Exception("p4rt-ctl: '{}' command requires at least {} "
                         "arguments".format(command, expected_nr))
 
+grpc_server_addr = ""
 
 def main():
+    global grpc_server_addr
     if len(sys.argv) < 2:
         print("p4rt-ctl: missing command name; use --help for help")
         sys.exit(1)
     parser = argparse.ArgumentParser(usage=USAGE)
-    parser.add_argument('command', help='Subcommand to run')
-
-    args = parser.parse_args(sys.argv[1:2])
-    if args.command not in all_commands.keys():
+    parser.add_argument('-g', '--grpc_addr', required=False, type=str, help="P4Runtime gRPC server address, format : <server IP>:<port>")
+    parser.add_argument('command', nargs='*', help='Subcommand to run')
+    args = parser.parse_args()
+    if args.grpc_addr is None:
+        grpc_server_addr = 'localhost:9559'
```

Comment:
If we set the default to the global variable as `localhost`, we could probably skip this `if` section and make the logic simpler to follow

Reply:
Default value is set to localhost:9559 for P4runtimeClient class - https://github.com/ipdk-io/networking-recipe/blob/3c6ed77af0786d0993a816ce5c2fedfff73f88de/clients/p4rt-ctl/p4rt-ctl.in#L657

Reply:
Ack